### DISCUSSION TITLE
Basic INSPIRE metadata implementation on dataset and series level

### DISCUSSION THREAD

**Discussion by MM-GIS** [INSPIRE-MIF/helpdesk#196] [Q&A] [answered]
Hi folks, 

We need to implement INSPIRE metadata in some future projects within my organization, but I start to get lost within the different versions, themes, etc. Our data deliverables are going to be related to Offshore Energy Resources, but we have the flexibility to decide ourselves the schema and the components we are going to use in our metadata, provided that they are INSPIRE compatible.

As we are trying to keep things as simple as possible, I assume it would be easier to adopt a more generic, simplified version of INSPIRE, rather than adopting a more complicated version of a specific theme. On the other hand, I am not sure if by not adopting a specific theme will actually make the schema template creation and metadata population easier, or not. 

The main format of deliverables will be Vector and Raster data, within ESRI File Geodatabases. So metadata will be provided on both Dataset and Series level. Is there any obligation that we have to comply with regarding the specific field of operation, data type and data level, or we can just adopt a generic INSPIRE v.2 model? And if yes, where I could find an example xml template schema that we could use for reference? The corresponding .xsd could also be useful though.

Thank you very much in advance for your answers!

**Comment by fabiovinci** [comment, marked as answer]
Dear @MM-GIS,

to create INSPIRE-compliant metadata you need to follow all the requirements contained in the related [Technical Guidance](https://github.com/INSPIRE-MIF/technical-guidelines/tree/main/metadata/metadata-iso19139).

The ISO XML metadata file can be validated using the [INSPIRE Reference Validator](https://inspire.ec.europa.eu/validator/test-selection/index.html).

In addition, you can find some sample metadata files [here](https://github.com/INSPIRE-MIF/helpdesk-validator/tree/master/examples) containing inline comments for each TG Requirement/Recommendation.

I hope this helps to clarify.
